hs-abci-server/src/Network/ABCI/Types/Messages/Common.hs

```diff
@@ -0,0 +1,9 @@
+module Network.ABCI.Types.Messages.Common
+  ( defaultABCIOptions
+  ) where
+
+import           Data.Aeson        (Options)
+import           Data.Aeson.Casing (aesonDrop, camelCase)
+
+defaultABCIOptions :: String -> Options
+defaultABCIOptions prefix = aesonDrop (length prefix) camelCase
```

Comment:
Let's instead use `snakeCase`, as i'm using this json instance for tendermint sdk bindings and in go people use snakeCase